src/common/modules/navigations/lnb/LNB.vue

```diff
@@ -31,7 +31,7 @@
                 </router-link>
             </div>
             <template v-for="(menuData, idx) in menuSet">
-                <l-n-b-menu-item :key="`${idx}-${getUUID()}`" :has-top-title="!!topTitle.label" :menu-data="menuData"
+                <l-n-b-menu-item :key="`${idx}-${getUUID()}`" :menu-data="menuData"
```

Comment:
추가로 `menuData`에 `id`가 있다면 `key` 바인딩을 `id`를 이용하면 어떨까요? 
`getUUID`는 매번 새로운 값을 리턴해서 뷰의 key 바인딩을 통해 취할 수 있는 이득을 완전 포기하는 형태라 아쉬워서요!
만일 id가 같은 데이터가 있어서 그렇다면, unique하다고 여겨지는 값을 함께 사용하는 것도 좋을 것 같아요.

```suggestion
  <l-n-b-menu-item v-for="(menuData, idx) in menuSet" :key="`${idx}-${menuData.id}-${menuData.label}`" :menu-data="menuData"
```

Reply:
Aㅏ 이 부분은 아직 조금 어려운게 menuData는 lnb menu item에서 한번 더 v-for로 반복이 돌아가는 형태라
현재는 array 형태예요! 그래서 바로 데이터를 꺼내올수가 없습니다 ㅠㅠ

Reply:
반복문을 위로 올리던가할게요!

Reply:
아니 반복문을 위로 올렸더니 scopedSlot 쓰는 방식까지 변경해야해서 ㅠㅠ 이거 제가 시간 갖고 좀 고민해볼게요!

Reply:
넵~